package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "c3",
+  "name": "alooma-c3",
```

Comment:
This will break everything if merged into master.

Reply:
i know i did this to publish to npm, i will change into a specific branch for merging for you